Libraries/Image/RCTImageLoader.m

```diff
@@ -716,10 +716,30 @@ - (RCTImageLoaderCancellationBlock)getImageSizeForURLRequest:(NSURLRequest *)ima
         CGSize size;
         if ([imageOrData isKindOfClass:[NSData class]]) {
             NSDictionary *meta = RCTGetImageMetadata(imageOrData);
-            size = (CGSize){
-                [meta[(id)kCGImagePropertyPixelWidth] doubleValue],
-                [meta[(id)kCGImagePropertyPixelHeight] doubleValue],
-            };
+
+            double imageOrientation = [meta[(id)kCGImagePropertyOrientation] doubleValue];
```

Comment:
We have to use `integerValue` method, `NSInteger` type and no conversion.

Reply:
oops, my bad. Fix it!